README.md

```diff
@@ -163,6 +163,12 @@ While it is not strictly necessary to enable the redis module, it is recommended
 - `docker-compose down`
 
   Destroys the containers (permanently deletes the state).
+  
+## Gulp
+
+Copy `.env.js` as `env.js` into the project root from [the Gulp component](https://github.com/Pronovix/gulp), then run
+`docker-compose run --rm gulp sh -c 'npm run gulp'` to build the CSS, or 
```

Comment:
Could you improve this part of the documentation? Something like:

`docker-compose run --rm gulp sh -c 'npm run gulp'` to build the CSS, or
`docker-compose run --rm gulp sh -c 'npm run gulp watch'` to continuously build the CSS when a change happens in `.scss` files.
You can use `--debug` to build the non-minified CSS with sourcemaps, e.g. `docker-compose run --rm gulp sh -c 'npm run gulp watch -- --debug'` (standalone `--` is needed to pass the argument to Gulp).